src/app/shared/directives/fresh-course.directive.ts

```diff
@@ -0,0 +1,23 @@
+import { Directive, ElementRef, Input } from '@angular/core';
+import { Course } from '../models/course.model';
+
+@Directive({
+  selector: '[appFreshCourse]'
+})
+export class FreshCourseDirective {
+  @Input('appFreshCourse') course: Course;
+
+  constructor(private el: ElementRef) {
+    let dateCreation = ''; // ERROR if: this.course.dateCreation.toDateString();
```

Comment:
If you are using type Date for `dateCreation` field, you may get ms for comparing `.getTime()` instead of parsing to string.